Fix builds on s12_91 & later, since configure picks up that libc has forkpty()
but doesn't realize we stuck the required definitions in <sys/termios.h>
instead of <pty.h> or <util.h>

Will be sent upstream after rebasing to current xscreensaver-5.34 release.

diff --git a/configure.in b/configure.in
index 182633b..1ad3e89 100644
--- a/configure.in
+++ b/configure.in
@@ -3285,7 +3285,7 @@ fi
 ###############################################################################
 
 PTY_LIBS=
-AC_CHECK_HEADERS(pty.h util.h)
+AC_CHECK_HEADERS(pty.h util.h sys/termios.h)
 AC_CHECK_X_LIB(util, forkpty,
                [PTY_LIBS="-lutil"
                 ac_have_forkpty=yes
diff --git a/hacks/apple2-main.c b/hacks/apple2-main.c
index b2fad16..a68f3ad 100644
--- a/hacks/apple2-main.c
+++ b/hacks/apple2-main.c
@@ -40,6 +40,9 @@
 # ifdef HAVE_UTIL_H
 #  include <util.h>
 # endif
+# ifdef HAVE_SYS_TERMIOS_H
+#  include <sys/termios.h>
+# endif
 #endif /* HAVE_FORKPTY */
 
 #include "screenhack.h"
diff --git a/hacks/phosphor.c b/hacks/phosphor.c
index 233acb6..bd01102 100644
--- a/hacks/phosphor.c
+++ b/hacks/phosphor.c
@@ -41,6 +41,9 @@
 # ifdef HAVE_UTIL_H
 #  include <util.h>
 # endif
+# ifdef HAVE_SYS_TERMIOS_H
+#  include <sys/termios.h>
+# endif
 #endif /* HAVE_FORKPTY */
 
 #include "screenhack.h"
diff --git a/hacks/xmatrix.c b/hacks/xmatrix.c
index 94c1b07..73fdf8b 100644
--- a/hacks/xmatrix.c
+++ b/hacks/xmatrix.c
@@ -91,6 +91,9 @@
 # ifdef HAVE_UTIL_H
 #  include <util.h>
 # endif
+# ifdef HAVE_SYS_TERMIOS_H
+#  include <sys/termios.h>
+# endif
 #endif /* HAVE_FORKPTY */
 
 #define CHAR_COLS 16
